app/views/users/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<% with_options scope: 'contribute.pages.users.new.form' do |i18n| %>
+  <%= simple_form_for @user, url: { action: @user.persisted? ? :update : :create } do |user| %>
+    <%= user.input(:email, label: i18n.t('labels.email')) %>
+    <%= user.input(:role, label: i18n.t('labels.role'), as: :select, collection: User.role_enum.dup) %>
```

Comment:
Why `.dup` the `User.role_enum` return value?

Reply:
no longer needed